Clarification of the customer's reports (Order Total, New, Order Count, Wish Lists, Segments) (#1221)
magento/merchdocs: src/reports/customer-reports.md

The New Accounts Report shows the number of new customer accounts opened during a specified time interval or date range.

### Workspace Controls

|Control|Description|
|--- |--- |

> |FromTo| Search by date range. Initiates a search for the accounts based on the start to end date.|

Reviewer: ```suggestion
|From / To|Used to define a search for the new accounts based on the start and end date.|
```

> |Show By|defines the granularity of the order record splitting. Options: Month/ Day/ Year |

Reviewer: ```suggestion
|Show By|Defines the granularity of the order record splitting. Options: Month / Day / Year |
```

|Refresh|Updates the grid to the specified filters.|
|Export|Exports the selected records as a CSV or Excel XML file.|

> |Scope| Displayed in which store the report will be generated.|

Reviewer: ```suggestion
|Scope|Used to set the site or store for which the report will be generated.|
```

> The Customer Wish List Report provides information about customers Wish Lists.

Reviewer: ```suggestion
The Customer Wish List Report provides information about customer Wish Lists.
```

> ### Workspace Controls

Reviewer: ```suggestion
### Workspace controls
```

|Control|Description|
|--- |--- |

> |Scope| Displayed in which store the report will be generated.|

Reviewer: ```suggestion
|Scope|Used to set the site or store for which the report will be generated.|
```

|Search| Initiates a search by the specified parameters.|
|Reset Filter| Initiates a reset of all search parameters.|

> |Per Page| Displays how many records are listed at the same time.|

Reviewer: ```suggestion
|Per Page| Sets the number of records displayed in a single page. |
```

|Export|Exports the selected records as a CSV or Excel XML file.|

> |FromTo|Search by date range. Initiates a search for the wishlists based on the start to end date.|

Reviewer: ```suggestion
|From / To|Used to define a search for the wish lists based on the start and end date.|
```

> |Wishlist| Initiates a wishlist search by name.|

Reviewer: Sadly, the UI is inconsistent with wish list as one or two words.

```suggestion
|Wishlist| Initiates a wish list search by name.|
```

> |Status| The current status in which the wishlist is displayed. Options: Privet/ Public |

Reviewer: ```suggestion
|Status| The current status of the wish list. Options: Private / Public |
```

> |Comment| Initiates a search by words in the wishlist comment.|

Reviewer: ```suggestion
|Comment| Initiates a search by text in the wish list comments.|
```

> ### Column description

Reviewer: ```suggestion
### Column descriptions
```

|Column|Description|
|--- |--- |

> |Added| Date of the wishlist creation.|

Reviewer: ```suggestion
|Added| Date the wish list was created.|
```

> |Customer| First and last name of the customer, who has created the wishlist.|

Reviewer: ```suggestion
|Customer| First and last name of the customer who created the wish list.|
```

> |Wishlist| Name of the wishlist.|

Reviewer: ```suggestion
|Wishlist| Name of the wish list.|
```

> |Status| The current status in which the wishlist is displayed. Options: Privet/ Public |

Reviewer: ```suggestion
|Status| The current status of the wish list. Options: Private / Public |
```

> |Product| Name of the product added to wishlist.|

Reviewer: ```suggestion
|Product| Name of the product added to the wish list.|
```

> |SKU| Sku of the product added to wishlist.|

Reviewer: ```suggestion
|SKU| SKU of the product added to the wish list.|
```

> |Comment| Сomment described upon wishlist creation.|

Reviewer: ```suggestion
|Comment| Сomment entered when the wish list was created.|
```

*Segments Report*

The Customer Segment Report provides information about the number of customers in each segment.

### Workspace Controls

|Control|Description|
|--- |--- |
|Search| Initiates a search by the specified parameters.|
|Reset Filter| Initiates a reset of all search parameters.|

> |Action| Initiates the display of segments by parameters. Options: Action/ View Combined Report|

Reviewer: ```suggestion
|Action| Initiates the display of segments by parameters. Options: Action / View Combined Report|
```

> |Per Page| Displays how many records are listed at the same time.|

Reviewer: |Per Page| Sets the number of records displayed in a single page. |
